parity/main.rs

```diff
@@ -495,15 +501,22 @@ fn execute_account_cli(conf: Configuration) {
 		println!("{:?}", new_address);
 		return;
 	}
+
 	if conf.args.cmd_list {
 		println!("Known addresses:");
-		for &(addr, _) in &secret_store.accounts().unwrap() {
+		for addr in &secret_store.accounts() {
 			println!("{:?}", addr);
 		}
 		return;
 	}
+
 	if conf.args.cmd_import {
-		let imported = util::keys::import_keys_paths(&mut secret_store, &conf.args.arg_path).unwrap();
+		let from = DiskDirectory::at(conf.keys_path());
+		let mut imported = 0;
+		for path in &conf.args.arg_path {
+			let to = DiskDirectory::at(path);
```

Comment:
you import in the wrong direction, don't you?
arg_path is the source for importing
and it supports both individual key files and directories with keys